src/swaggerConfig.ts

```diff
@@ -0,0 +1,15 @@
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+
+function addSwaggerConfig(args) {
```

Comment:
Rename args to app and add types for app and return type void.

Reply:
ok